domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/DefendantTimeline.java

```diff
@@ -1,28 +1,39 @@
 package uk.gov.hmcts.cmc.domain.models;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
+import javax.validation.Valid;
 import javax.validation.constraints.Size;
 
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
-public class DefendantTimeline extends Timeline {
+public class DefendantTimeline {
+
+    @Valid
+    @Size(max = 20)
+    @JsonProperty("rows")
+    private final List<TimelineEvent> events;
 
     @Size(max = 99000)
     private final String comment;
 
     public DefendantTimeline(List<TimelineEvent> events, String comment) {
-        super(events);
+        this.events = events;
         this.comment = comment;
     }
 
     public Optional<String> getComment() {
         return Optional.ofNullable(comment);
     }
 
+    public Optional<List<TimelineEvent>> getEvents() {
```

Comment:
I am not convinced to that, it is good pattern to always have list initialised, if so there is no need to use optional for lists.